README.md

```diff
@@ -15,6 +15,12 @@ development & testing:
 
     oc create -f deploy/examples/ceph-demo.yaml
 
+> If your cluster runs multiple compute nodes you need to allow TCP traffic to
+> the Ceph pod on port 6800-7300. Please have a look at
```

Comment:
How are we running the containers in the pods?  Because in the Ember-CSI demo the Ember-CSI container and the Ceph container are running with the host network, so I think they have access by default, unless you add extra security.

Reply:
This is at least needed on openshift nodes, because the network is locked down pretty much and by default blocking traffic to the Ceph pod. Since this is also working on a range the kube proxy can't handle this - so needed

Reply:
> This is at least needed on openshift nodes, because the network is locked down pretty much and by default blocking traffic to the Ceph pod. Since this is also working on a range the kube proxy can't handle this - so needed

Well, need to check this again - maybe we can limit this to two ports, and in that case get rid of the iptables requirement. Checking

Reply:
I'm OK with the current addition, I don't think changes are necessary.